api/server/sdk/server_interceptors.go

```diff
@@ -99,7 +99,7 @@ func (s *sdkGrpcServer) auth(ctx context.Context) (context.Context, error) {
 	// Determine issuer
 	issuer, err := auth.TokenIssuer(token)
 	if err != nil {
-		return nil, auditLogWarningf(codes.Unauthenticated, "Unable to obtain token issuer from authorization token: %v", err)
```

Comment:
why are you removing the error value here?

Reply:
This was just throwing the error into the logs, which could have a token in it.